src/app/tests/suites/certification/Test_TC_S_2_2.yaml

```diff
@@ -645,6 +748,34 @@ tests:
               - name: "SceneList"
                 value: [0x01]
 
+    - label:
+          "Step 8d: TH sends a AddScene command to DUT with the GroupID field
+          set to G1, the SceneID field set to 0x01, the TransitionTime field set
+          to 70 000 000 (70 000s) and no extension field sets. This should fail
```

Comment:
test plan suggestion - try adding right at the border (6000000) as well, and maybe right over the border (6000001)

Reply:
Created issue to add those in test plan here: https://github.com/project-chip/connectedhomeip/issues/31897.
Added in Yaml.